### PR TITLE
harden comparison, add @throws annotations

### PR DESCRIPTION
**Pull request by vlakarados**
Adds missing `@throws` annotations

### CODE REVIEW

File: lib/Injector.php [hunk 1]
```diff
@@ -97,6 +97,7 @@ public function defineParam($paramName, $value)
      *
      * @param string $original The typehint to replace
      * @param string $alias The implementation name
+     * @throws ConfigException if any argument is empty or not a strings
```
Comment (kelunik) [inline lib/Injector.php]: s/strings/string

**Comment by vlakarados** [comment]
I don't think I get it.. Why has travis failed?

**Comment by kelunik** [comment]
Because of a PHP 7 change, where reflection functions can not be bound to any scope anymore. /cc @bwoebi @nikic

**Comment by kelunik** [comment]
I think this was fixed in PHP `master`, can you restart the build on Travis? I can't. Maybe close and reopen helps.

**Comment by bwoebi** [comment]
Nope, @rdlowrey has to restart it.Anyway, impossible this PR has broken anything ;-)

**Comment by vlakarados** [comment]
Yep, there's only one comparison change in code.

**Pull request merged**